### PR TITLE
Archer - fix details concat bug + offset bug

### PR DESCRIPTION
**Pull request by guykeller**
## Status 
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/19592
fixes: https://github.com/demisto/etc/issues/18463

## Description
Fixing two fetch incidents bugs:
1. Details of different incidents were  concatenated, now they aren't.Solution tested by customer and works.
2. Fetch was unstable and would sometimes fail due to a problem with the offset. Solution tested by customer and works.

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by guykeller** [comment]
@kirbles19 FYI - just the second point of the CHANGELOG was not approved by you yet.

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -2948,16 +2948,18 @@ script:
         if (params.timeZone === '0') {
             tz = 0;
         }
-        var offset = tz ? (60 * 1000 * new Date().getTimezoneOffset()) : 0;
-        offset = tz ? parseInt(tz) : 0;
+
+        offset = tz ? 60 * 1000 * parseInt(tz) : 60 * 1000 * new Date().getTimezoneOffset();
```
Comment (ronykoz) [inline Integrations/integration-ArcherRSA.yml]: add a comment explaining the 60 * 1000

Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: Fixed

File: Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -2081,6 +2081,7 @@ script:
 
         if (records && records.Contents && records.Contents.length > 0) {
             for (var i = 0; i < records.Contents.length; i++) {
+                details = "";
```
Comment (hirshbergshachar) [inline Integrations/integration-ArcherRSA.yml]: @guykeller is this used for anything? If not, please remove

**Comment by kirbles19** [comment]
@guykeller Updated all RNs. Done.

File: Integrations/integration-ArcherRSA.yml (hunk 2 above)
Comment (guykeller) [reply Integrations/integration-ArcherRSA.yml]: This is what fixes the concatenation issue.
As you can see, the "details" variable is declared outside the scope of the for loop, thus isn't re-written for each record.
This fixes it.

**Pull request merged**